MekHQ/src/mekhq/campaign/parts/MissingSpacecraftEngine.java

```diff
@@ -79,12 +78,12 @@ public double getTonnage() {
     public boolean isAcceptableReplacement(Part part, boolean refit) {
         return part instanceof SpacecraftEngine
                 && getName().equals(part.getName())
-                && getTonnage() == ((SpacecraftEngine)part).getTonnage();
+                && getTonnage() == part.getTonnage();
```

Comment:
I assume SpacecraftEngine doesn't have an implementation of getTonnage()?

Reply:
Same as above